### PR TITLE
VTCCA-4511: updated error focus for date

### PR DESCRIPTION
**Pull request by zahiriqbal**

### CODE REVIEW

File: app/views/propertyLinking/ownershipToProperty.scala.html [hunk 1]
```diff
@@ -55,7 +55,7 @@
 
 @getEnrichedErrorLinks = @{
     toCustomDateFormErrors.toSet.map{ error: FormError =>
-        ErrorLink(href = Some(s"#${error.key}"),
+        ErrorLink(href = Some(s"#${error.key}.day"),
```
Comment (darrell-plant) [inline app/views/propertyLinking/ownershipToProperty.scala.html:58]: why `day`? (what about month and year?)

Comment (zahiriqbal) [reply app/views/propertyLinking/ownershipToProperty.scala.html:58]: @darrell.plant I spoke to @will.blakeney and he said only Day field should be focused, as the error summary message is only one and doesn't break down into day, month and year.

Comment (darrell-plant) [reply app/views/propertyLinking/ownershipToProperty.scala.html:58]: yep, I was just puzzled, but makes sense

Comment (darrell-plant) [reply app/views/propertyLinking/ownershipToProperty.scala.html:58]: and before adding `day` it didn't work - got it now :)

File: app/views/propertyLinking/ownershipToProperty.scala.html [hunk 2]
```diff
@@ -126,7 +126,7 @@
                 item =>
                     InputItem(
                         id = s"toDate.$item",
-                        classes = s"govuk-input--width-${if(item == "year") "4" else "2"}",
+                        classes = s"govuk-input--width-${if(item == "year") "4" else "2"}${if(form(s"toDate.$item").hasErrors || form.errors.exists(_.key.isEmpty)) " govuk-input--error" else ""}",
```
Comment (darrell-plant) [inline app/views/propertyLinking/ownershipToProperty.scala.html:129]: this 
```
s"govuk-input--width-${if(item == "year") "4" else "2"}${if(form(s"toDate.$item").hasErrors || form.errors.exists(_.key.isEmpty)) " govuk-input--error" else ""}"
```
is repeated, so could be extracted to method, but not a biggie

**Comment by PeterPerhac** [comment]
👍  looks good to me

**Pull request merged**